Makefile

```
@@ -543,29 +545,32 @@ tools/install: \
-update: \
-	clean-generated \
-	update/libs \
-	update/actions \
-	proto/all \
-	deps \
-	update/template \
-	go/deps \
-	go/example/deps \
-	rust/deps \
-	format
+update:
```

checkmake (0.2.2)

[warning] Target body for "update" exceeds allowed length of 5 (10).

(maxbodylength)